### PR TITLE
[ Applciation ] PicoGPT Android Applciation with NNTrainer @open sesame 10/12 16:10

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR includes the PicoGPT(https://github.com/jaymody/picoGPT) Android Application with NNTrainer.
We only use the PicoGPT Model Binary and provides the NNTrainer implementation #2212. This is the Android application implementation for that PR.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by myungjoo** [comment]
Please add required files only. Do not add auto generated files.

**Review by lhs8928 — APPROVED** [review]
LGTM

File: Applications/Android/PicoGPTJNI/.gradle/buildOutputCleanup/buildOutputCleanup.lock
Comment (myungjoo) [inline Applications/Android/PicoGPTJNI/.gradle/buildOutputCleanup/buildOutputCleanup.lock:1]: Do you really seriously need to add this file? (and all the other auto-generated files)

**Review by Hyoungsukim — APPROVED** [review]
lgtm

File: Applications/Android/PicoGPTJNI/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Android NNtrainer Applicaiton Sample
```
Comment (djeong20) [inline Applications/Android/PicoGPTJNI/README.md:2]:
```suggestion
title: Android NNtrainer Application Sample
```

**Pull request merged**